qiskit_metal/_gui/main_window.py

```diff
@@ -127,25 +127,37 @@ def delete_all_components(self):
             self.gui.refresh()
 
     @slot_catch_error()
-    def save_design_as(self, _=None):
-        """Handles click on Save Design As."""
+    def save_design_copy(self):
+        """Saves a separate copy of design under a different name"""
         filename = QFileDialog.getSaveFileName(
             None,
             'Select a new location to save Metal design to',
-            self.design.get_design_name() + '.metal',
-            selectedFilter='*.metal')[0]
+            self.design.get_design_name() + '.py',
```

Comment:
Suggestion of naming convention:

python <filename>.metal.py

Reply:
I saw updated message below.  Can you add to the warning which explains  that there are unique integer component ids and unique names within Metal back-end.  If the script is pasted in the jupyter notebook and run, the component ids  could be different than than the initial use of Metal.  This should not impact the second notebook.  

The different sequence would happen for two reasons.  In the second notebook, if there were components already added to the  notebook before the script is pasted and run, then Metal components would have sequential ids, which are different than the first notebook.  The second scenario is when user deletes a component in the first notebook, there would be a "hole" in the sequence of component ids.  When the script is run in second notebook, there would not be a "hole" in the sequence.  


![image](https://user-images.githubusercontent.com/74020801/124605372-d4d28280-de39-11eb-890c-8e583ff1b415.png)